app/controllers/issue_templates_settings_controller.rb

```diff
@@ -1,26 +1,25 @@
 class IssueTemplatesSettingsController < ApplicationController
   unloadable
   before_filter :find_project, :find_user
-  before_filter :authorize, :except => [ :show_help, :preview ]
+  before_filter :authorize, except: [ :show_help, :preview ]
 
   def edit
     if params[:settings] != nil
       @issue_templates_setting = IssueTemplateSetting.find_or_create(@project.id)
       attribute = params[:settings]
-      @issue_templates_setting.update_attributes(:enabled => attribute[:enabled],
-                                                 :help_message => attribute[:help_message],
-                                                 :inherit_templates => attribute[:inherit_templates],
-                                                 :should_replaced => attribute[:should_replaced])
+      @issue_templates_setting.update_attributes(enabled: attribute[:enabled],
+                                                 help_message: attribute[:help_message],
+                                                 inherit_templates: attribute[:inherit_templates],
+                                                 should_replaced: attribute[:should_replaced])

```

Comment:
__[rubocop]__
* Line is too long. [94/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)